src/styles/main.scss

```diff
@@ -30,6 +30,11 @@ a {
     text-decoration: none;
 }
 
+.text-content {
```

Comment:
I'm trying to figure out what this is trying to achieve and why it's in main.scss — it looks like you only use it once above, and it's actually added as a class by JavaScript?